MiniF/notifications/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+from django.conf import settings
+from profiles.models import StartupProfile, InvestorProfile
+
+
+class NotificationType(models.Model):
+    """
+    Model for type of notification
+    """
+    type = models.CharField(max_length=100, unique=True)
```

Comment:
!!!
Problems / comments:

type is a built-in Python name (object type name), it is better to avoid such names to avoid conflicts.

It is better to use, for example, name:

    name = models.CharField(max_length=100, unique=True)

Then the __str__ method should also be changed to:

    def __str__(self):
        return self.name